CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 ## 1.11.0 / 2021-06-07
 
-* [ENHANCEMENT] API client: Add newer fields to Rules API #855
-* [ENHANCEMENT] API client: Add missing fields to Targets API #856
-* [FEATURE] API client: Add exemplars API support #861
-* [CHANGE] Add new collectors package #862
+* [CHANGE] Add new collectors package. #862
+* [CHANGE] `prometheus.NewExpvarCollector` use `collectors.NewExpvarCollector` instead. #862
+* [CHANGE] `prometheus.NewGoCollector` use `collectors.NewGoCollector` instead. #862
+* [CHANGE] `prometheus.NewBuildInfoCollector` use `collectors.NewBuildInfoCollector` instead. #862
```

Comment:
Now it sounds even more dramatic. (o:

Perhaps make clear that it is merely deprecated, not removed? Something like "Deprecated `prometheus.NewGoCollector`, use `collectors.NewGoCollector` instead."

Reply:
That was what I've meant 😅 I just missed the deprecated statement. Downsides of working in a park :D